guides/common/modules/proc_rex-pull-based-transport-tuning.adoc

```diff
@@ -0,0 +1,33 @@
+[id="REX_pull_based_transport_tuning_{context}"]
+= REX pull-based transport tuning
+
+{Project} has a pull-based transport that has the features of REX but with a deployment model similar to katello-agent. 
+This transport uses MQTT as its messaging protocol and includes an MQTT client running on each host.
+In order to tune the MQTT server (`mosquitto`) and increase the number of hosts connected to it, you can follow the following procedure on the {ProjectServer}:
+
+.Procedure
+
+. Enable pull-based remote execution on your {ProjectServer} or {SmartProxyServer}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+{foreman-installer} --foreman-proxy-plugin-remote-execution-script-mode pull-mqtt
+----
++
+Note that your {ProjectServer} or {SmartProxyServer} can only use one transport mode, either SSH or MQTT.
+. To increase the default number of hosts accepted by the MQTT server, create the following system unit definition file and set its contents as specified in the below given example (this will allow the `mosquitto` service to handle ~5000 hosts):
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+cat >/etc/systemd/system/mosquitto.service.d/limits.conf <<__EOF__
```

Comment:
I think this one is invalid because the installer will purge this file.

And even if it is, I think we need an issue to track removal of this workaround. https://github.com/eclipse/mosquitto/issues/2674 isn't getting any traction so if this is needed, an installer issue to manage this is needed.